packages/plugin-loyalties-ui/src/automations/automation.tsx

```
@@ -32,6 +33,16 @@ const Automations = props => {
     const { result } = props;
     return <>{result}</>;
   }
+
+  if (componentType === 'triggerForm') {
+    const [_serviceName, contentType] = activeTrigger?.type.split(':');
```

Biome (1.9.4)

[error] Unsafe usage of optional chaining.

If it short-circuits with 'undefined' the evaluation will throw TypeError here:


(lint/correctness/noUnsafeOptionalChaining)